lldb/include/lldb/Target/Target.h

```diff
@@ -1193,11 +1193,14 @@ class Target : public std::enable_shared_from_this<Target>,
   // 2 - if there is a process, then read from memory
   // 3 - if there is no process, then read from the file cache
   //
+  // The optional did_read_live_memory parameter will be set true if live memory
+  // was read successfully.
+  //
   // The method is virtual for mocking in the unit tests.
   virtual size_t ReadMemory(const Address &addr, void *dst, size_t dst_len,
                             Status &error, bool force_live_memory = false,
-                            lldb::addr_t *load_addr_ptr = nullptr);
-
+                            lldb::addr_t *load_addr_ptr = nullptr,
+                            bool *did_read_live_memory = nullptr);
```

Comment:
Could you add a marker indicating that this is a swift-specific modification of this API? It's important to keep track of these things.

Reply:
I'm upstreaming this.